composer.json

```diff
@@ -5,12 +5,12 @@
     "type": "symfony-bundle",
     "require": {
         "php": ">=5.5",
-        "symfony/symfony": "~2.5",
+        "symfony/symfony": "3.1.*",
```

Comment:
No changes were made for the bundle code, so why is the version changed to 3.1?, If you had no issues, or deprecation warnings, then this bundle can support both ~2.5 & ~3.0, so the rule should be something like `"symfony/symfony": "~2.5|~3.0"`